checks/evaluation/security_policy.go

```diff
@@ -64,9 +64,20 @@ func SecurityPolicy(name string, findings []finding.Finding) checker.CheckResult
 			e := sce.WithMessage(sce.ErrScorecardInternal, "score calculation problem")
 			return checker.CreateRuntimeErrorResult(name, e)
 		}
+
+		// Log all findings.
+		checker.LogFindings(findings, nil, nil, dl)
 		return checker.CreateMinScoreResult(name, "security policy file not detected")
 	}
 
+	if score == checker.MaxResultScore {
+		// Log all findings except the negative ones.
+		checker.LogFindings(findings, nil, []finding.Outcome{finding.OutcomeNegative}, dl)
+	} else {
+		// Log all findings.
+		checker.LogFindings(findings, nil, nil, dl)
+	}
+
```

Comment:
does this check have any situations where there would be negative findings but max score?
can this be simplified to just logging findings?

Reply:
> does this check have any situations where there would be negative findings but max score?

I don't think so.

> can this be simplified to just logging findings?

wdum?

Reply:
> wdum?

I'm saying we can probably get rid of the if/else and conditional filtering and just have one instance of
```
checker.LogFindings(findings, dl)
```